core/kazoo_amqp/src/api/kapi_dialplan.hrl

```diff
@@ -595,6 +595,16 @@
                               ]).
 -define(PLAY_STOP_REQ_TYPES, []).
 
+%% PlaySeek Request
+-define(PLAY_SEEK_REQ_HEADERS, [<<"Application-Name">>, <<"Call-ID">>, <<"Duration">>, <<"Direction">>]).
+-define(OPTIONAL_PLAY_SEEK_REQ_HEADERS, [<<"Insert-At">>]).
+-define(PLAY_SEEK_REQ_VALUES, [{<<"Event-Category">>, <<"call">>}
+                              ,{<<"Event-Name">>, <<"command">>}
+                              ,{<<"Application-Name">>, <<"playseek">>}
+                              ,{<<"Insert-At">>, <<"now">>}
+                              ]).
+-define(PLAY_SEEK_REQ_TYPES, []).
```

Comment:
I would like `Duration` to have an `fun is_integer/1` type validator.

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)